README.rst

```diff
@@ -1,69 +1,105 @@
+API. The **nidaqmx** package does not require installation of the C header
+files.
 
 Some functions in the **nidaqmx** package may be unavailable with earlier
-versions of the NI-DAQmx driver. Visit the
-`ni.com/downloads <http://www.ni.com/downloads/>`_ to upgrade your version of
-NI-DAQmx.
+versions of the NI-DAQmx driver. Refer to the Installation section for details
+on how to install the latest version of the NI-DAQmx driver.
+
+Operating System Support
+------------------------
 
 **nidaqmx** supports Windows and Linux operating systems where the NI-DAQmx
 driver is supported. Refer to
 `NI Hardware and Operating System Compatibility <https://www.ni.com/r/hw-support>`_
 for which versions of the driver support your hardware on a given operating
 system.
 
+Python Version Support
+----------------------
+
 **nidaqmx** supports CPython 3.8+ and PyPy3.
 
 Installation
 ============
 
-Running **nidaqmx** requires NI-DAQmx to be installed. Visit
-`ni.com/downloads <http://www.ni.com/downloads/>`_ to download the latest
-version of NI-DAQmx. None of the recommended **Additional items** are required
-for **nidaqmx** to function, and they can be removed to minimize installation
-size. It is recommended you continue to install the **NI Certificates** package
-to allow your Operating System to trust NI built binaries, improving your
-software and hardware installation experience.
-
-**nidaqmx** can be installed with `pip <http://pypi.python.org/pypi/pip>`_::
+**nidaqmx** can be installed with `pip <http://pypi.python.org/pypi/pip>`_ from
+`pypi <https://pypi.org/project/nidaqmx/>`_::
 
   $ python -m pip install nidaqmx
 
-Similar Packages
-================
+Automatic Driver Installation
+-----------------------------
 
-There are similar packages available that also provide NI-DAQmx functionality in
-Python:
+Running **nidaqmx** requires NI-DAQmx to be installed. The **nidaqmx** module
+ships with a command-line interface (CLI) to streamline the installation
+experience. You can install the NI-DAQmx driver using the following command::
 
-- `daqmx <https://pypi.org/project/daqmx/>`_
-  (`slightlynybbled/daqmx on GitHub <https://github.com/slightlynybbled/daqmx>`_)
-  provides an abstraction of NI-DAQmx in the ``ni`` module.
+  $ python -m nidaqmx installdriver
 
-- PyLibNIDAQmx (`pearu/pylibnidaqmx on GitHub <https://github.com/pearu/pylibnidaqmx>`_)
-  provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
-  with this package's module name.
+On Windows, this command will download an online streaming installer from
+ni.com. On Linux, this will download the repository registration package for
+your Linux distribution and install the driver using your package manager. For
+more details on what versions are being installed, refer to the
+`_installer_metadata.json <https://github.com/ni/nidaqmx-python/blob/master/generated/nidaqmx/_installer_metadata.json>`_
+on GitHub that is distributed with the **nidaqmx** package.
```

Comment:
If it's distributed with the nidaqmx package wouldn't it be better for users to look at the local copy? The version on GitHub may not match the version they are using.

Doesn't the installdriver command say what version it's going to install?

Reply:
It should, but looking at the code, I'm not sure if it does. I'll create an issue for that and resolve it before we make a minor release to update the metadata. I'll remove this blurb.